langchain/src/client/index.ts

```diff
@@ -1,6 +1 @@
-export {
-  LangChainPlusClient,
-  Dataset,
-  Example,
-  DatasetRunResults,
-} from "./langchainplus.js";
+export { DatasetRunResults, runOnDataset } from "./langchainplus.js";
```

Comment:
Do we actually need this export at all anymore?

Reply:
Or just for backwards compat?

Reply:
Specifically for importing from the index? Are we discouraging importing from the index for ALL directories or just the ones with optional dependencies?